internal/compiler/lookup.rs

```diff
@@ -461,24 +471,44 @@ impl LookupObject for ElementRc {
     }
 }
 
+pub enum StyleMetricsPropertyUse {
+    Acceptable,
+    Deprecated(String),
+    Unacceptable,
+}
+
+/// Returns Some() if name is a deprecated style metrics property.
+/// Returns None if the property is acceptable for regular use.
+/// Returns Err(()) if the property should not be accessible.
```

Comment:
You're commenting the old behavior :-)

Reply:
Oooops. I had it return Result in an earlier attempt and indeed forgot to adjust the docs. Will fix tomorrow.